frontend/src/libs/components/table/table.tsx

```diff
@@ -90,37 +122,75 @@ const Table = <T,>({
     </thead>
   );
 
+  const getHandler = (
+    index: number,
+    arrayLength: number,
+  ): typeof onEditClick | typeof onDeleteClick => {
+    if (!isTableEditable) {
+      return;
+    }
+    const { isEditCell } = checkIsIconCell(isTableEditable, index, arrayLength);
```

Comment:
maybe let's better rename checkIsActionCell? for me sounds better, and don't forget about renaming the file